index.html

```diff
@@ -341,6 +344,25 @@ <h2 class="primary-header-text color-deep-blue portfolio-heading">
         </div>
     </footer>
 
+    <section id="mobile-menu" class="mobile-menu-background display-none">
+        <div class="menu-mobile-div">
+            <h6 class="close-menu toggle-mobile-menu" >X</h6>
```

Comment:
- [ ] The hamburger and closed button must follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins). how about using a closed menu class icon, for example: `<i class="fa fa-close"></i>` instead of using a `h6` tag with an `x` ? for more documentation you can check this [link](https://www.w3schools.com/icons/tryit.asp?filename=tryicons_fa-close) or you can download icon image from template Figma design to make sure you are using the same icon with the original design

Reply:
@BenMukebo I have added ```h6``` here because linters were giving a **warning: no header in a section**, so thought to add so, if you suggest ignoring the warning and replace with a font-awesome icon, we can replicate the same.
Please guide me on the same.

Reply:
@RitobrotoMukherjee, why not add the `header` tag?